tds-itemselection-common/src/main/java/tds/itemselection/base/ItemGroup.java

```diff
@@ -115,7 +114,7 @@ public void setSegmentPosition (long _segmentPosition) {
    * @return the _numRequired
    */
   public int getNumRequired () {
-    return numberOfItemsRequired;
+    return numberOfItemsRequired == null ? 0 : numberOfItemsRequired;
```

Comment:
`numberOfItemsRequired` is an Integer so when Java tries to autobox this it'll throw an NPE if it is null.  I chose to just handle it in this getter because there are getters for the Integer object and I didn't want to add a subtle bug when something is looking for a null `numberOfItemsRequired`.